src/components/dialog/dialog.js

```diff
@@ -661,10 +685,12 @@ function MdDialogProvider($$interimElementProvider) {
       /**
        * Hide modal backdrop element...
        */
-      options.hideBackdrop = function hideBackdrop() {
+      options.hideBackdrop = function hideBackdrop($destroy) {
```

Comment:
should be `options.$destroy`; used to indicate a **skip** of hide backdrop animations.